### ISSUE TITLE
Availability on ROS2 Foxy

### ISSUE THREAD

**Issue by Enes1097** [ros-perception/image_pipeline#690]
Hello,
I was just about to build this package in ROS2 Foxy, but realized that there is no branch referencing the Foxy distribution yet.
Is this package also going to be released in this distribution?

Thank you for your answer!

**Comment by JWhitleyWork** [comment]
Version 2.2.1 was released for Foxy and is available as binaries on the ROS repositories. This release was created from the `ros2` branch. However, now that Galactic is out, I will create a `foxy` branch and release for Galactic from the `ros2` branch.

**Comment by Enes1097** [comment]
Thank you for your quick answer!
Before I asked this question I already tried building the package thinking that the "ROS2" branch is refering to the foxy distribution, but I did not work. This is what my shell gave as an output.

![Screenshot from 2021-08-11 17-28-49](https://user-images.githubusercontent.com/57830794/129180917-7771cf94-8bcb-4377-a0c9-7527fb3317ef.png)
![Screenshot from 2021-08-11 17-38-14](https://user-images.githubusercontent.com/57830794/129180925-17247711-4940-4c7d-8c47-51719017ab30.png)

How is the package refering to a _catkin_ file, if the package is in ROS2 and not ROS1?

(I'm sorry, I just started working with ROS, so I'm a beginner)

**Comment by Enes1097** [comment]
Thank you for your quick answer!
Before I asked this question I already tried building the package thinking that the "ROS2" branch is refering to the foxy distribution, but I did not work. This is what my shell gave as an output.

![Screenshot from 2021-08-11 17-28-49](https://user-images.githubusercontent.com/57830794/129184275-96d63c15-4f53-4b14-acab-fd3267a76067.png)
![Screenshot from 2021-08-11 17-38-14](https://user-images.githubusercontent.com/57830794/129184278-b92521a4-8598-4416-a541-26054345021f.png)


How is the package refering to a catkin file, if the package is in ROS2 and not ROS1?

(I'm sorry, I just started working with ROS, so I'm a beginner)
